.github/workflows/build_test.yml

```diff
@@ -39,4 +42,4 @@ jobs:
       - name: Build
         run: ninja -C build
       - name: Test
-        run: ctest --test-dir build --output-on-failure
+        run: ctest --test-dir build --output-on-failure
```

Comment:
Missing line at the end of file.

Reply:
We probably want to put this in another PR but here's a workflow that enforces files end in newlines:

```
 - name: All files end with newline
 - run: find dir1 dir2 -type f | xargs -L1 bash -c 'test "$(tail -c1 "$0")" && echo "No newline at end of $0"'
```

Here's the article:
https://medium.com/@alexey.inkin/how-to-force-newline-at-end-of-files-and-why-you-should-do-it-fdf76d1d090e

Reply:
I will added to list